ihaskell-display/ihaskell-widgets/src/IHaskell/Display/Widgets/Button.hs

```diff
@@ -0,0 +1,210 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module IHaskell.Display.Widgets.Button (
+    -- * The Button Widget
+    Button,
+    -- * Predefined button styles
+    ButtonStyle(..),
+    -- * Create a new button
+    mkButton,
+    -- * Set button properties
+    setButtonStyle,
+    setButtonLabel,
+    setButtonTooltip,
+    disableButton,
```

Comment:
It seems more consistent to use `set`, as in `setButtonEnabled`. So instead of `enableButton`, `disableButton`, and `isDisabled`, switch to `setButtonEnabled` and `getButtonEnabled`.

Not for now, but when we build a bigger API on top of this, maybe considering creating lenses for these and exporting them would work somehow.